src/app/private/[uid]/chapter-leader/seniors/layout.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import Sidebar, { ISideBar } from "@components/Sidebar";
+import {
+  faEnvelope,
+  faHome,
+  faHouseLock,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
+import React, { useContext } from "react";
+import { UserContext } from "src/context/UserProvider";
+
+interface IAdminLayout {
+  children: React.ReactNode;
+}
+
+const AdminLayout = ({ children }: IAdminLayout) => {
+  const { user } = useContext(UserContext);
+  const buttons: ISideBar["buttons"] = React.useMemo(
+    () => [
+      {
+        name: "Home",
+        link: `/private/${user.id}/admin/home`,
```

Comment:
Double check the routings here. I think it should be `chapter-leader` instead of admin right?